platform/solr/solr-factory/src/main/java/org/codice/solr/factory/SolrClientFactory.java

```diff
@@ -30,5 +30,16 @@
    * @return the newly created {@code SolrClient}
    * @throws IllegalArgumentException if <code>core</code> is <code>null</code>
    */
-  public SolrClient newClient(String core);
+  SolrClient newClient(String core);
+
+  SolrClient newClient(String collection, String alias, SolrCollectionConfiguration configuration);
```

Comment:
:question: Is there a reason `alias` is not part of the `SolrCollectionConfiguration` instead?

Reply:
alias is a property of splitCatalogProvider that ties all collectionProvider together.

Reply:
Since it is specific to the split provider implementation, that makes me think it should not be in either interface.  I wonder if we need to add a map of properties to the `SolrCollectionConfiguration` similar to what we had to do with the Catalog requests.